### PR TITLE
Feature regression tests

### PR DESCRIPTION
**Pull request by avalanche123**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -42,7 +42,7 @@ Vagrant.configure("2") do |config|
   echo "Compiling and installing the extension..."
   pushd /usr/local/src/php-driver/ext
   phpize
-  LDFLAGS="-L$builddir/lib" LIBS="-lssl -lz -luv -lgmp -lstdc++" ./configure --with-cassandra=/tmp/php-driver-installation/build --with-gmp=/tmp/php-driver-installation/build --with-libdir=lib
+  LDFLAGS="-L$builddir/lib" LIBS="-lssl -lz -luv -lm -lgmp -lstdc++" ./configure --with-cassandra=/tmp/php-driver-installation/build --with-gmp=/tmp/php-driver-installation/build --with-libdir=lib
```
Comment (avalanche123) [inline Vagrantfile:45]: why do we depend on -lm?

File: support/ccm.php [hunk 2]
```diff
@@ -94,16 +94,22 @@ public function stop()
...
     }
 
-    public function setup()
+    private function getClusters()
     {
         $clusters = array();
-        foreach (explode("\n", $this->run('list')) as $cluster) {
-            if ($this->isActive($cluster)) {
-                $active = count($clusters);
+        foreach (explode("\r\n", $this->run('list')) as $cluster) {
```
Comment (avalanche123) [inline support/ccm.php]: let's use `PHP_EOL` instead of `"\r\n"` here.

File: features/bootstrap/FeatureContext.php [hunk 3]
```diff
@@ -155,6 +157,41 @@ public function itsOutputShouldContain(PyStringNode $string)
         PHPUnit_Framework_Assert::assertContains((string) $string, $this->getOutput());
     }
 
+    /**
+     * @Given the following logger settings:
+     */
+    public function theFollowingLoggerSettings(PyStringNode $string)
+    {
+        $lines = preg_split("/\\r\\n|\\r|\\n/", $string->getRaw());
```
Comment (avalanche123) [inline features/bootstrap/FeatureContext.php:315]: you can use `explode(PHP_EOL, $string->getRaw())` here

File: Vagrantfile (hunk 1 above)
Comment (mikefero) [inline Vagrantfile:45]: Logarithmic functions are being used (https://github.com/datastax/php-driver/blob/master/ext/util/math.c#L427) and were not being linked in (e.g. undefined references)

File: support/ccm.php (hunk 2 above)
Comment (mikefero) [inline support/ccm.php]: Good catch

File: features/bootstrap/FeatureContext.php (hunk 3 above)
Comment (mikefero) [inline features/bootstrap/FeatureContext.php:315]: Tried that and the result was one line returned from explode when on Windows platform. I can continue to play around with other options.

File: Vagrantfile (hunk 1 above)
Comment (avalanche123) [inline Vagrantfile:45]: gotcha, thanks!

File: features/bootstrap/FeatureContext.php (hunk 3 above)
Comment (avalanche123) [inline features/bootstrap/FeatureContext.php:315]: nah, leave whatever works in

**Pull request merged**